modules/api-workbench-create.adoc

```diff
@@ -0,0 +1,25 @@
+:_module-type: PROCEDURE
+
+[id='api-workbench-create_{context}']
+= Creating a workbench by using the OpenShift AI API
+
+[role="_abstract"]
+Add descriptive text here
+
+.Prerequisites
+* one
+* two
+
+.Procedure
+
+. Step 1
+. Step 2
+
+.Verification
+
+. Step 1
+. Step 2
+
+.Additional resources
```

Comment:
Do we need to include the following line before the `.Additional resources` line?

`[role='_additional-resources']`

Also, should those lines be commented out by default?